src/aws_encryption_sdk/streaming_client.py

```diff
@@ -202,24 +202,28 @@ def readable(self):
         # Open streams are currently always readable.
         return not self.closed
 
-    def read(self, b=None):
+    def read(self, b=-1):
```

Comment:
Curious, why change from None to -1?

Reply:
PEP3116

https://www.python.org/dev/peps/pep-3116/